README.md

```diff
@@ -70,7 +70,7 @@ Additional Resources
 --------------------
 
 + [Contributing patches](https://maven.apache.org/guides/development/guide-maven-development.html#Creating_and_submitting_a_patch)
-+ [Apache Maven Install JIRA project page][jira]
++ [Apache Maven Install JIRA project page](https://issues.apache.org/jira/projects/MINSTALL/issues)
```

Comment:
should be removed at all - project use GitHub issues

Reply:
There are still enough open issues there to make this useful.